### PR TITLE
Use JS SDK instead of iframe 

### PR DESCRIPTION
**Pull request by roosnic1**
WPC-215. This PR improves the polis implementation and uses the Javascript SDK instead of an iframe.

### CODE REVIEW

**Review by penk85 — APPROVED** [review]
:+1:

**Review by HanaAyman — APPROVED** [review]

File: packages/editor/src/client/blocks/embeds/polis.tsx [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-      src={`https://pol.is/${conversationID}`}
-      width="100%"
-      height="600"
-      scrolling="yes"
-      frameBorder="0"
-    />
+    <div
+      className="polis"
+      data-page_id={conversationID}
+      data-site_id="polis_site_id_ssFrYaCCx4UpbUX63X"></div>
```
Comment (bishoymelek) [inline packages/editor/src/client/blocks/embeds/polis.tsx:17]: what is this id? should be static? and how does this attribute work?

Comment (roosnic1) [reply packages/editor/src/client/blocks/embeds/polis.tsx]: Good catch. I changed it

**Review by penk85 — APPROVED** [review]

**Review by HanaAyman — APPROVED** [review]

**Review by bishoymelek — APPROVED** [review]
:tada:

**Pull request merged**